Framework/PythonInterface/plugins/algorithms/LoadWANDSCD.py

```diff
@@ -48,46 +117,106 @@ def validateInputs(self):
         return issues
 
     def PyExec(self):
-        runs = self.getProperty("Filename").value
-
-        if not runs:
+        # Process input data
+        # get the list of input filenames
+        runs = self.get_intput_filenames()
+        # load and group
+        data = self.load_and_group(runs)
+
+        # Process the vanadium normalization data
+        if self.getProperty("NormalizedBy").value.lower() == 'none':
+            # no normalization
+            self.getLogger().information("Skip Va normalization as requested")
```

Comment:
I think they still want the ability to normalize the data with the vanadium but not do any additional scaling. `'none' == scale=1`  So the `self.normalize` should always run if a vanadium workspace is provided.